src/Docfx.Plugins/MarkdownServiceParameters.cs

```diff
@@ -35,6 +35,13 @@ public class MarkdownServiceProperties
     [JsonProperty("alerts")]
     [JsonPropertyName("alerts")]
     public Dictionary<string, string> Alerts { get; set; }
+
+    /// <summary>
+    /// PlantUml extension configuration parameters
+    /// </summary>
+    [JsonProperty("plantUml")]
+    [JsonPropertyName("plantUml")]
+    public Dictionary<string, string> PlantUml { get; set; }
```

Comment:
Can this be turned into a strongly-typed object so we know what properties to expect for PlantUml? The `alerts` is a string dictionary because alerts maps from name to css class.

Reply:
I would also prefer going this route, but the problem is that the configuration is owned by the extension and there is no reference to the extension from within the Plugins assembly, so there is no opportunity to use a strongly typed object.